recipes/zlib-ng/all/conanfile.py

```diff
@@ -85,9 +86,12 @@ def package_info(self):
         suffix = "" if self.options.zlib_compat else "-ng"
         self.cpp_info.names["pkg_config"] = "zlib" + suffix
         if self.settings.os == "Windows":
-            static_flag = "static" if not self.options.shared and tools.Version(self.version) >= "2.0.5" else ""
+            # The library name of zlib-ng is complicated in zlib-ng>=2.0.4:
+            # https://github.com/zlib-ng/zlib-ng/blob/2.0.4/CMakeLists.txt#L994-L1016
```

Comment:
omg, why does it need to be so complicated? 🤦

Reply:
especially changing it in a minor version...

Reply:
In this commit, upstream decides to change naming rule in MINGW, MSYS and CYGWIN.
https://github.com/zlib-ng/zlib-ng/commit/2c65dce35f0e109a6e05413730bfe698c8bf6407

We can simplify this after removing 2.0.2.